frontend/src/components/pages/Landing.tsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import { useHistory } from "react-router-dom";
+import {
+  Box,
+  Center,
+  Text,
+  Stack,
+  VStack,
+  Image,
+  Button,
+  Link,
+} from "@chakra-ui/react";
+import * as Routes from "../../constants/Routes";
+
+import JumpMathLogo from "../../assets/jump-math-logo.png";
+
+const PageLinks = (): React.ReactElement => {
+  const history = useHistory();
+  const navigateTo = () => history.push(Routes.LOGIN_PAGE);
+
+  return (
+    <VStack gap={1.5}>
+      <Button onClick={navigateTo} width={300} variant="primary">
```

Comment:
```suggestion
      <Button onClick={navigateTo} width={{base: "10em", md: "20em"}} variant="primary">
```
responsive width for the 3 buttons

Reply:
oooo nice, ty!